### PR TITLE
Sample/native extension klarna checkout

### PR DESCRIPTION
**Pull request by bullfroggy-sap**

### CODE REVIEW

File: native-extension/klarna/checkout/klarna-checkout/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# KlarnaCheckout
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 11.2.14.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
```
Comment (justinlee01) [inline native-extension/klarna/checkout/klarna-checkout/README.md:7]: I believe these commands are from the Angular CLI. Do you want to replace them with `npm start`, `npm run build`, etc?

**Comment by justinlee01** [comment]
Why is the directory structure /klarna/checkout/klarna-checkout? Will there be future files in /klarna and /klarna/checkout?

File: native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(KlarnaCheckoutComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```
Comment (justinlee01) [inline native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts:23]: I think we should unit test this. Isn't that part of SAP's guidelines for publishing things?

Comment (bullfroggy-sap) [reply native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts:23]: > Why is the directory structure /klarna/checkout/klarna-checkout? Will there be future files in /klarna and /klarna/checkout?

I went with Richard's suggestion

Comment (bullfroggy-sap) [reply native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts:23]: > I think we should unit test this. Isn't that part of SAP's guidelines for publishing things?

Maybe. We would probably need to setup some kind of pipeline.

File: native-extension/klarna/checkout/klarna-checkout/README.md (hunk 1 above)
Comment (bullfroggy-sap) [reply native-extension/klarna/checkout/klarna-checkout/README.md:7]: This is a generated file which should be replaced by Shaun's readme

File: native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts (hunk 2 above)
Comment (justinlee01) [reply native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts:23]: What was the reasoning for Richard's suggestion?

Comment (bullfroggy-sap) [reply native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts:23]: I don't know, frankly I don't think it matters. This is not the permanent location for this

Comment (justinlee01) [reply native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts:23]: Frankly I do think it matters, unless you create a ticket to move it to its correct place. This is for public consumption.

**Review by justinlee01 — APPROVED** [review]
Well, if this is strictly used as a sample, it looks good.

**Pull request merged**